packages/ember-toucan-form/package.json

```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "@babel/runtime": "^7.20.7",
     "@embroider/addon-shim": "^1.0.0",
+    "ember-async-data": "^1.0.3",
```

Comment:
I don't think this needs to be a `dependency`. We are not actively using it, only ember-headless-form does and that has it already as a dependency. With the import being changes to a type-only import (see below), this can be a devDependency I think!

Reply:
done